build.gradle.kts

```diff
@@ -31,6 +31,8 @@ dependencies {
 
     // Provides the tools to load the system configuration
     api("com.typesafe:config:1.4.1")
+    // scireum parsii
+    api("com.scireum:parsii:5.0.1")
```

Comment:
Glaub die kann jetzt wieder raus wenn der aktuelle develop reingemerged wurde

Reply:
stimmt, guter Punkt. Das ist ja mittlerweile auch in kernel gewandert